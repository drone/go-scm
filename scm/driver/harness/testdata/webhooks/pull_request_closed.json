{
  "trigger": "pullreq_closed",
  "repo": {
    "id": 22,
    "path": "codeowners/asdsad",
    "uid": "asdsad",
    "default_branch": "main",
    "git_url": "http://localhost:3000/git/codeowners/asdsad.git"
  },
  "principal": {
    "id": 3,
    "uid": "admin",
    "display_name": "Administrator",
    "email": "admin@gitness.io",
    "type": "user",
    "created": 1696332021613,
    "updated": 1696332021613
  },
  "pull_req": {
    "number": 6,
    "state": "closed",
    "is_draft": false,
    "title": "Create sad",
    "source_repo_id": 22,
    "source_branch": "asdxsa",
    "target_repo_id": 22,
    "target_branch": "main",
    "author": {
      "id": 3,
      "uid": "admin",
      "display_name": "Administrator",
      "email": "admin@gitness.io",
      "type": "user",
      "created": 1696332021613,
      "updated": 1696332021613
    },
    "pr_url": "http://localhost:3000/codeowners/asdsad/pulls/14"
  },
  "target_ref": {
    "name": "refs/heads/main",
    "repo": {
      "id": 22,
      "path": "codeowners/asdsad",
      "uid": "asdsad",
      "default_branch": "main",
      "git_url": "http://localhost:3000/git/codeowners/asdsad.git"
    }
  },
  "ref": {
    "name": "refs/heads/asdxsa",
    "repo": {
      "id": 22,
      "path": "codeowners/asdsad",
      "uid": "asdsad",
      "default_branch": "main",
      "git_url": "http://localhost:3000/git/codeowners/asdsad.git"
    }
  },
  "sha": "27822dd2ec788f924c97b0b194c5bfd906f2e574",
  "commit": {
    "sha": "27822dd2ec788f924c97b0b194c5bfd906f2e574",
    "message": "",
    "author": {
      "identity": {
        "name": "Administrator",
        "email": "admin@gitness.io"
      },
      "when": "2023-12-20T13:10:47+05:30"
    },
    "committer": {
      "identity": {
        "name": "Gitness",
        "email": "system@gitness.io"
      },
      "when": "2023-12-20T13:10:47+05:30"
    }
  },
  "head_commit": {
    "sha": "27822dd2ec788f924c97b0b194c5bfd906f2e574",
    "message": "",
    "author": {
      "identity": {
        "name": "Administrator",
        "email": "admin@gitness.io"
      },
      "when": "2023-12-20T13:10:47+05:30"
    },
    "committer": {
      "identity": {
        "name": "Gitness",
        "email": "system@gitness.io"
      },
      "when": "2023-12-20T13:10:47+05:30"
    }
  }
}