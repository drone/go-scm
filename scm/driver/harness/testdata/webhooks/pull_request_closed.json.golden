{
  "Action": "closed",
  "Repo": {
    "ID": "22",
    "Namespace": "",
    "Name": "asdsad",
    "Branch": "main",
    "Private": false,
    "Clone": "http://localhost:3000/git/codeowners/asdsad.git",
    "CloneSSH": "",
    "Link": "http://localhost:3000/git/codeowners/asdsad.git",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "PullRequest": {
    "Number": 6,
    "Title": "Create sad",
    "Body": "",
    "Sha": "27822dd2ec788f924c97b0b194c5bfd906f2e574",
    "Ref": "refs/heads/asdxsa",
    "Source": "asdxsa",
    "Target": "main",
    "Fork": "fork",
    "Link": "http://localhost:3000/codeowners/asdsad/pulls/14",
    "Closed": true,
    "Merged": false,
    "Author": {
      "ID": "admin",
      "Login": "admin",
      "Name": "Administrator",
      "Email": "admin@gitness.io",
      "Avatar": "",
      "Created": "2023-10-03T16:50:21.613+05:30",
      "Updated": "2023-10-03T16:50:21.613+05:30"
    },
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "Sender": {
    "ID": "admin",
    "Login": "admin",
    "Name": "Administrator",
    "Email": "admin@gitness.io",
    "Avatar": "",
    "Created": "2023-10-03T16:50:21.613+05:30",
    "Updated": "2023-10-03T16:50:21.613+05:30"
  }
}